eZ/Publish/API/Repository/Tests/Stubs/RepositoryStub.php

```diff
@@ -632,6 +659,24 @@ public function rollback()
         }
 
         --$this->transactionDepth;
+        unset( $this->commitEventsQueue[$this->transactionCount] );
```

Comment:
It's not really relevant anymore :)
*signals are now by default sent only when transaction they are in are committed, making them all transaction safe and ready for being async safe.*